systemvm/patches/debian/vpn/etc/ipsec.d/l2tp.conf

```diff
@@ -30,4 +35,5 @@ conn L2TP-PSK
         # ----------------------------------------------------------
         # Change 'ignore' to 'add' to enable this configuration.
         #
+        rightsubnetwithin=10.1.2.0/8
```

Comment:
Having hardcoded 10.0.0.0/8 cannot work on remote management VPN. Because if the remote client internal subnet is in the 10.0.0.0/8 the VPN will fail to update routes on the client side.

Why not have /24 hardcoded but based on the ip range defined in the global settings?  
so if user set  172.16.0.10 - 18  it will be using 172.16.0.0/24? Or the cleanest method would be to use the smallest available subnet but this would require more code I guest. in this example the subnet would be set to 172.16.0.0/27